### DISCUSSION TITLE
🤗 Space to Quants EXL2?

### DISCUSSION THREAD

**Discussion by Katehuuh** [turboderp/exllamav2#568] [Ideas]
Example:
- [🤗gguf-my-repo](https://huggingface.co/spaces/ggml-org/gguf-my-repo)
- [🤗mergekit-gui](https://huggingface.co/spaces/arcee-ai/mergekit-gui)

**Comment by Anthonyg5005** [comment]
Currently there's an exllama server with a bot (@blockblockblock on hf), the creators of the bot are also working on a website version. I also have scripts to automate the quanting process in the terminal a bit more but I could also try connecting it to a gradio, not sure it can really be connected to a space though as exl2 quants take hours to process
